test/templates/fake_files/config-override.erb

```diff
@@ -0,0 +1,9 @@
+<%-
+config.rules[:misc] << 'runtime'
+config.rules[:ignore] << 'Gemfile'
+-%>
```

Comment:
Not sure if this would not deserve some comment about the expected behavior, because for my first look, it took me several minutes to understand the code.

But since I don't have specific idea/suggestion, this is certainly not show stopper ;)

Reply:
I added short comments for every line. Does that help?